HelloInput/Class1.cs

```diff
@@ -5,7 +5,6 @@
 using System.Linq;
 using System.Text;
 using System.Threading.Tasks;
-using System.Windows.Forms;
 
 namespace HelloInput
```

Comment:
It is highly recommended to create one file for one class. I see two classes in one file. Can you please break it down to two files? Also, name the files accordingly. For e.g. SquaredSumActivity.cs and SimpleSumActivity.cs

Reply:
> * Do you need to check in .suo file to this repo?
> * I don't think you need to check in files under bin and obj folder? Can you add unwanted files/folders to .gitignore? https://www.freecodecamp.org/news/gitignore-what-is-it-and-how-to-add-to-repo/
Could you please have a look and see if my gitignore syntax is correct? I've tried every way I can think of and have not been able to exclude these files/folders
> * Another thing I would like you to figure out is how to add unit tests to these activities?
Added an executable solution called UnitTests

Reply:
> It is highly recommended to create one file for one class. I see two classes in one file. Can you please break it down to two files? Also, name the files accordingly. For e.g. SquaredSumActivity.cs and SimpleSumActivity.cs

Done